src/api/quantum/meta/Universal.java

```diff
@@ -0,0 +1,13 @@
+package api.quantum.meta;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Represents a universal object
+ */
+@Target({ElementType.TYPE, ElementType.CONSTRUCTOR, ElementType.METHOD, ElementType.PACKAGE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Universal {}
```

Comment:
@Spartan322 It's universal between sides. Will just merge the universal class to a side enum in a commit later.